db/migrate/20220125052845_fix_topic_like_count_including_whispers.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class FixTopicLikeCountIncludingWhispers < ActiveRecord::Migration[6.0]
+  def up
```

Comment:
We've already ran this migration before as I copied this from https://github.com/discourse/discourse/blob/5dd8b827e867254a5c621c9a6415233daf0cbade/db/migrate/20200707154522_fix_topic_like_count.rb. 

However I have chosen to run this migration irregardless of whether whisper is currently enabled or not as a site could have disabled whispers after they have enabled it.